utils/types.go

```
@@ -295,6 +296,13 @@ func GetKonnectClient(httpClient *http.Client, config KonnectConfig) (*konnect.C
 		httpClient = http.DefaultClient
 		httpClient.Transport = defaultTransport
 	}
+	if config.ProxyAddress != "" {
+		proxyURL, err := url.Parse(config.ProxyAddress)
+		if err == nil {
```

Can you please return an error in case `err` is not nil?